fe/fe-core/src/main/java/org/apache/doris/analysis/ColumnDef.java

```diff
@@ -283,11 +285,15 @@ public static void validateDefaultValue(Type type, String defaultValue) throws A
             case CHAR:
             case VARCHAR:
             case HLL:
-            case STRING:
                 if (defaultValue.length() > scalarType.getLength()) {
                     throw new AnalysisException("Default value is too long: " + defaultValue);
                 }
                 break;
+            case STRING:
```

Comment:
if StringType length is MAX_STRING_LENGTH, your change is the same as before.
if we set StringType length while create table, such as set to 100, the default value length should not bigger than 100, not MAX_STRING_LENGTH.

Reply:
The length cannot be specified when creating a string type field. The default is the maximum length

Reply:
The length cannot be specified when creating a string type field. The default is the maximum length. 

This is also to avoid modifying the allowable length of string type in other places. If you debug your own local code, you will find that the maximum allowable length of string has become 1.